### DISCUSSION TITLE
WED V1.3 Wall polygons unknown/height offset really unknown?

### DISCUSSION THREAD

**Discussion by Incrementis** [NearInfinityBrowser/NearInfinity#180] [Q&A] [answered]
Good evening,

When I studied the **AR2606.WED** file, especially the wall polygons, I found that there is an inconsistency in **Near Infnity** and **iesdp** regarding the naming of an offset.

In **Near Infinity** the offset is named as **unknown**(see screenshot below):
![grafik](https://github.com/NearInfinityBrowser/NearInfinity/assets/17119474/4acea80d-2b47-422b-b3f5-73a7311dd84b)
While in **iesdp** it is named as **Height**(see screenshot below):
![grafik](https://github.com/NearInfinityBrowser/NearInfinity/assets/17119474/45d55c03-7279-4c14-9c6a-7c80e4f1788f)

Here is the link to the **iesdp** WED documentation: https://gibberlings3.github.io/iesdp/file_formats/ie_formats/wed_v1.3.htm 

I don't know what this offset does as I couldn't find any more information in **iesdp**, but the fact that it has a "comprehensible" name does mean that naming the offset in **Near Infnity* * is wrong and should be fixed?
Since I'm not an expert, I posted these name differences here in the Questions and Answers section instead of posting it as a ticket.

**Comment by Argent77** [comment, marked as answer]
Good catch!

I don't know what `Height` actually does, but I can certainly add it to NI.
